requirements.txt

```diff
@@ -1 +1,9 @@
+black==22.6.0
+coverage[toml]==6.4.4
+mypy==0.971
+pylint==2.7.1
+pymdown-extensions==9.5
+pyspelling==2.8.1
+pytest==6.2.5
+typed-argument-parser==1.7.2
```

Comment:
all the dependencies needed for testing are already included in requirements_qa.txt
please, discard this change
if you believe that the original requirements.txt misses something crucial, tell us in the chat, students are not supposed to modify this file in this lab

Reply:
done